heudiconv/convert.py

```diff
@@ -115,7 +115,8 @@ def prep_conversion(sid, dicoms, outdir, heuristic, converter, anon_sid,
 
     # if conversion table(s) do not exist -- we need to prepare them
     # (the *prepare* stage in https://github.com/nipy/heudiconv/issues/134)
-    reuse_conversion_table = op.exists(edit_file)
+    # if overwrite - recalculate this anyways
+    reuse_conversion_table = op.exists(edit_file) and not overwrite
```

Comment:
I just feel we need a dedicated option to either reuse conversation table, otherwise we again keep pulling rope two ways back and forth... Eg what if someone might want to rerun conversion with new dcm2niix using his tuned conversion table?

Reply:
Perhaps a new option would be the best solution.. I ran into this problem when debugging a new heuristic, looks like we came across this bridge previously further down in the code :)

https://github.com/nipy/heudiconv/blob/9c3e82379ab70931b10e3b93bf96c3e30cb1d75a/heudiconv/convert.py#L128-L135

Reply:
@yarikoptic how do you feel about forcing regeneration of the conversion table if the heuristic has changed?

Reply:
Ambivalent ;)  but isn't what we are doing already with above check based on file_md5sum?